mkdocs/docs/HPC/linux-tutorial/common_pitfalls.md

```diff
@@ -49,6 +49,29 @@ But, this is tedious, and you can prevent errors by simply colouring
 within the lines and not using spaces in filenames.
 
 ### Missing/mistyped environment variables
+
+A common pitfall is the (accidental) use of non-defined variables.
+Contrary to what you may expect, this does *not* result in error
+messages, but the variable is considered to be *empty* instead.
+
+This may lead to surprising results, for example:
+<pre><code>$ <b>export WORKDIR=/tmp/test</b>
```

Comment:
not sure that I understand this example

Reply:
It seems like I moved this block of text from the original `getting_started.md` to `common_pitfalls.md` without reading it too well. This example indeed does not make any sense.

Reply:
solved in [a540e97](https://github.com/hpcugent/vsc_user_docs/pull/647/commits/a540e9753c0df5118ac0b3dd838743ecf75c1630)